src/ActionsMenuItem.js

```diff
@@ -17,43 +22,60 @@ type ActionsMenuItemProps = {
 export class ActionsMenuItem extends React.Component< ActionsMenuItemProps, {} > {
     render () {
         // We don't use FormattedMessage as we are working with a complex chain of templates.
-        const commandName = this.props.intl.formatMessage({ id: `Command.${this.props.itemKey}` });
+        let commandName = this.props.intl.formatMessage({ id: `ActionsMenuItem.command.${this.props.itemKey}` });
+        const usedLabel = this.props.intl.formatMessage({ id: 'ActionsMenuItem.usedItemToggleLabel' });
+        if (this.props.isUsed) {
+            commandName += " " + usedLabel;
+        }
+
         const commandNameShort = this.props.intl.formatMessage({ id: `Command.short.${this.props.itemKey}` });
+        const checkboxId = "actions-menu-item-" + commandNameShort.replace(/ /g, "");
 
-        const actionNameKey = this.props.isAllowed ? "ActionsMenu.item.action.show" : "ActionsMenu.item.action.hide";
+        const actionNameKey = this.props.isAllowed ? "ActionsMenuItem.action.show" : "ActionsMenuItem.action.hide";
         const actionName = this.props.intl.formatMessage({ id: actionNameKey });
 
-        // If we're used, show one message. If we're not, show another that differs based on `isAllowed`.
-        const showHideLabelKey = this.props.isUsed ? "ActionsMenu.item.usedItemToggleLabel" : "ActionsMenu.item.unusedItemToggleLabel";
-        const showHideLabel = this.props.intl.formatMessage(
-            { id: showHideLabelKey },
-            { action: actionName, commandName: commandName }
-        );
-
-        const showHideAriaLabelKey = this.props.isUsed ? "ActionsMenu.item.usedItemToggleAriaLabel" : "ActionsMenu.item.unusedItemToggleAriaLabel";
+        const showHideAriaLabelKey = this.props.isUsed ? "ActionsMenuItem.usedItemToggleAriaLabel" : "ActionsMenuItem.unusedItemToggleAriaLabel";
```

Comment:
With the changes to make simplification options check/uncheckable when they are used, we don't need to use "usedItemToggleAriaLabel" I think, as used option can be hidden.

Reply:
Fixed.